client-common/pom.xml

```diff
@@ -20,19 +20,19 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>com.cloudera.livy</groupId>
-    <artifactId>livy-main</artifactId>
+    <artifactId>livy-main_2.10</artifactId>
     <version>0.3.0-SNAPSHOT</version>
   </parent>
 
   <groupId>com.cloudera.livy</groupId>
-  <artifactId>livy-client-common</artifactId>
+  <artifactId>livy-client-common_2.10</artifactId>
```

Comment:
Do we need to add Scala binary version suffix like `2.10` to this module that is purely Java code? I checked Spark code, it adds suffix for such modules like `launcher`, `network-comm`.